code/modules/admin/topic.dm

```diff
@@ -1055,6 +1055,22 @@
 		to_chat(H, span_adminnotice("Your prayers have been answered!! You received the <b>best [new_item.name]!</b>"), confidential = TRUE)
 		SEND_SOUND(H, sound('sound/effects/pray_chaplain.ogg'))
 
+	else if (href_list["adminpopup"])
+		if (!check_rights(R_ADMIN))
+			return
+
+		var/message = input(owner, "As well as a popup, they'll also be sent a message to reply to. What do you want that to be?", "Message") as text|null
+		if (!message)
+			to_chat(owner, span_notice("Popup cancelled."))
+			return
+
+		var/client/target = locate(href_list["adminpopup"])
+		if (!istype(target))
```

Comment:
```suggestion
		if (QDELETED(target))
```
`!istype()` is generally legacy from before widespread qdel usage

Reply:
As per Discord

![image](https://user-images.githubusercontent.com/35135081/131269939-ba58d0a6-5f36-41c5-8b00-63050251f9b2.png)